src/app/features/public-form/public-form.component.scss

```diff
@@ -32,3 +32,13 @@ button {
   min-height: 60vh;
   text-align: center;
 }
+
+.form-validation-error {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
you can use our standard css classes from https://github.com/Aam-Digital/ndb-core/blob/master/src/styles/globals/_flex-classes.scss

I prefer those because the html already shows me immediately readable what will happen (e.g. "ah, this is a flex row that is align and has "margin-regular" using our standard sizes")